src/server/worker/client.go

```diff
@@ -126,7 +126,7 @@ func NewClient(address string) (Client, error) {
 	if err != nil {
 		return Client{}, err
 	}
-	conn, err := grpc.Dial(fmt.Sprintf("%s:%d", address, port),
+	conn, err := grpc.Dial(fmt.Sprintf("dns:///%s:%d", address, port),
```

Comment:
Likewise, i don't think we want this here

Reply:
Agree. The same argument above. Use the underlying system DNS by default.